src/confluent_kafka/admin/__init__.py

```diff
@@ -363,6 +363,22 @@ def delete_topics(self, topics, **kwargs):
 
         return futmap
 
+    def list_topics(self, **kwargs):
+        """
```

Comment:
I wonder if we can grab the doc-string from the parent class, to avoid duplication.
Maybe something like:
```python
    __doc__ = super(AdminClient, self).list_topics.__doc__
```